src/View/Helper/UrlHelper.php

```diff
@@ -71,6 +71,10 @@ public function build($url = null, $options = false)
      *   `fullBase` Return full URL with domain name
      *   `pathPrefix` Path prefix for relative URLs
      *   `plugin` False value will prevent parsing path as a plugin
+     *   `assetTimestamp` Overrides the value of `Asset.timestamp` in Configure.
```

Comment:
Given that all methods are for assets perhaps a shorter name `timestamp` would be enough.